src/main/java/io/fabric8/jenkins/openshiftsync/CredentialsUtils.java

```diff
@@ -274,10 +273,9 @@ private static Credentials secretToCredentials(Secret secret) {
     private static Credentials newSSHUserCredential(String secretName,
             String username, String sshKeyData) {
         if (secretName == null || secretName.length() == 0 ||
-                username == null || username.length()== 0 ||
                 sshKeyData == null || sshKeyData.length() == 0) {
             logger.log(Level.WARNING, "Invalid secret data, secretName: " +
-                secretName + " username: " + username + " sshKeyData: " +
+                secretName + " sshKeyData: " +
```

Comment:
i wonder if logging the secret key values is a bad idea from a security perspective...

Reply:
yeah probably ... aborting merge, will update